rest/model/distro.go

```diff
@@ -70,7 +72,50 @@ func (s *APIPlannerSettings) ToService() (interface{}, error) {
 	return interface{}(settings), nil
 }
 
+////////////////////////////////////////////////////////////////////////////////
+//
+// APIFinderSettings is the model to be returned by the API whenever distro.FinderSettings are fetched
+
+type APIFinderSettings struct {
+	Version APIString `json:"version"`
+}
+
+// BuildFromService converts from service level distro.FinderSettings to an APIFinderSettings
+func (s *APIFinderSettings) BuildFromService(h interface{}) error {
+	var settings distro.FinderSettings
+	switch v := h.(type) {
+	case distro.FinderSettings:
+		settings = v
+	case *distro.FinderSettings:
+		settings = *v
+	default:
+		return errors.Errorf("%T is not an supported expansion type", h)
+	}
+
+	if len(settings.Version) == 0 {
+		s.Version = ToAPIString(evergreen.FinderVersionLegacy)
+	} else {
+		s.Version = ToAPIString(settings.Version)
+	}
+
+	return nil
+}
+
+// ToService returns a service layer distro.FinderSettings using the data from APIFinderSettings
+func (s *APIFinderSettings) ToService() (interface{}, error) {
+	settings := distro.FinderSettings{}
+	settings.Version = FromAPIString(s.Version)
+	if len(settings.Version) == 0 {
```

Comment:
nit: if `settings.Version == ""` (checking the length means it's less obvious that this is a string, and Go programmers have strong expectations about checking the zero values of primitive types)

Reply:
Fixing